src/main/java/org/mule/service/http/impl/service/client/GrizzlyHttpClient.java

```diff
@@ -90,6 +92,9 @@
   private static final int DEFAULT_SEND_AND_DEFER_BUFFER_SIZE = KB.toBytes(10);
   private static final String DEFAULT_DECOMPRESS_PROPERTY_NAME = SYSTEM_PROPERTY_PREFIX + "http.client.decompress";
 
+  private static final String DISABLE_REQUEST_STREAMING_PROPERTY_NAME = SYSTEM_PROPERTY_PREFIX + "http.client.decompress";
```

Comment:
Fix the property name, it's the decompress one. For it's name, you can check out the response streaming property name in the 3.8.x branch